apps/cli/src/prebuild.ts

```diff
@@ -43,13 +51,55 @@ type RemixRoutes = {
   }>;
 };
 
+export const downloadAsset = async (
+  url: string,
+  name: string,
+  temporaryDir: string
+) => {
+  const assetPath = join("public", ASSETS_BASE, name);
+  const tempAssetPath = join(temporaryDir, name);
+  try {
+    await access(assetPath);
+  } catch {
+    try {
+      const response = await new Promise<IncomingMessage>((resolve, reject) => {
+        https
+          .get(url, (response) => {
+            if (response.statusCode !== 200) {
+              reject(new Error(`HTTP Error: ${response.statusCode}`));
+              response.resume();
+            }
+            resolve(response);
+          })
+          .on("error", (error) => {
+            reject(error);
+          });
+      });
```

Comment:
I meant specifically `fetch()`. It's available in node globally since v18.

Reply:
yea something likee this

```suggestion
fetch(url).then(res => new Promise((resolve, reject) => {
    const dest = fs.createWriteStream(fn);
    res.body.pipe(dest);
    res.body.on('end', () => resolve());
    dest.on('error', reject);
}));
```

Reply:
```js
await pipeline(response.body, fs.createWriteStream(path))
```